javascript/index.js

```diff
@@ -24,17 +24,123 @@
     document.querySelector("#mashedPotatoes").innerHTML += `<li>${step5}</li>`;
     document.querySelector("#mashedPotatoesImg").removeAttribute("hidden");
   }, (error) => console.log(error));
-
+ */
 
 
 // Iteration 1 - using callbacks
 // ...
 
+getInstruction('mashedPotatoes', 0, (step0) => {
+  document.querySelector("#mashedPotatoes").innerHTML += `<li>${step0}</li>`
+  getInstruction('mashedPotatoes', 1, (step1) => {
+    document.querySelector("#mashedPotatoes").innerHTML += `<li>${step1}</li>`
+    getInstruction('mashedPotatoes', 2, (step2) => {
+      document.querySelector("#mashedPotatoes").innerHTML += `<li>${step2}</li>`
+      getInstruction('mashedPotatoes', 3, (step3) => {
+        document.querySelector("#mashedPotatoes").innerHTML += `<li>${step3}</li>`
+        getInstruction('mashedPotatoes', 4, (step4) => {
+          document.querySelector("#mashedPotatoes").innerHTML += `<li>${step4}</li>`
+          getInstruction('mashedPotatoes', 5, (step5) => {
```

Comment:
Your mashed potatoes image is not displaying because you're calling getInstructions on step 5 and since there is no step 5 you will get an error and the rest of the code will not run